src/context/LocaleContext.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { createContext, useContext } from 'react'
+import { LocaleData, locales } from 'utils/locales'
+
+export type LocaleContext = {
+  currentLocale: LocaleData
+}
+
+export const LocaleContext = createContext<LocaleContext>({
+  currentLocale: locales['se-en'],
+})
+
+export const useLocale = () => {
+  const context = useContext(LocaleContext)
+  if (context === undefined) {
+    throw new Error('useLocale has to be used inside a LocaleProvider')
+  }
```

Comment:
Ah I thought you meant that I should set the default so we could remove it from the test. :) Because the test didn't fail now when we have a `defaultvalue` funny enough. But let's put it back. And sure, we can remove the check 🙌